### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by WillDignazio** [bencentra/bootstrap-csh#12]
We don't use the hacker green anymore, stuff like that.

**Comment by bencentra** [comment]
Yeah, if we aren't even using the green for the public site anymore there ins't any need for the green theme. Update the README and delete all the "public" theme files.
